skops/io/_utils.py

```diff
@@ -4,7 +4,7 @@
 import sys
 from dataclasses import dataclass, field
 from functools import singledispatch
-from typing import Any
+from typing import Any, List
```

Comment:
Just a comment, doesn't need to be changed since we don't do it consistently in the code base: With `from __future__ import annotations`, you can use `list[str]` etc., no need to import `List`.